gframe/game.cpp

```diff
@@ -1691,8 +1691,11 @@ void Game::PopulateSettingsWindow() {
 		gSettings.chkConfirmDeckClear = env->addCheckBox(gGameConfig->confirm_clear_deck, GetNextRect(), sPanel, CHECKBOX_CONFIRM_DECK_CLEAR, gDataManager->GetSysString(12104).data());
 		defaultStrings.emplace_back(gSettings.chkConfirmDeckClear, 12104);
 		gSettings.chkIgnoreDeckContents = env->addCheckBox(gGameConfig->ignoreDeckContents, GetNextRect(), sPanel, CHECKBOX_IGNORE_DECK_CONTENTS, gDataManager->GetSysString(12119).data());
-		menuHandler.MakeElementSynchronized(gSettings.chkIgnoreDeckContents);
 		defaultStrings.emplace_back(gSettings.chkIgnoreDeckContents, 12119);
+		// Todo: sysString
```

Comment:
For the string, you can use `12123`

Reply:
I get '???' for the sysString, is there something else i have to do?

Reply:
The id i gave you was a free spot not yet used by other strings, to have it show locally you'll have to manually add the string to your strings.conf file

Reply:
But how to add it so it gets distributed?